bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/immutable/Semester.scala

```diff
@@ -65,8 +65,8 @@ case class Semester(year: Int, half: SemesterOption) {
     cal.setTimeZone(TimeZone.getTimeZone("UTC"))
     cal.set(Calendar.YEAR, year)
     cal.set(Calendar.MONTH, half.firstMonth)
-    cal.set(Calendar.DAY_OF_MONTH, 0)
-    cal.set(Calendar.HOUR, 0)
+    cal.set(Calendar.DAY_OF_MONTH, 1)
+    cal.set(Calendar.HOUR_OF_DAY, 0)
```

Comment:
@andrewwstephens  The core semester [gets the time right](https://github.com/gemini-hlsw/ocs/blob/develop/bundle/edu.gemini.spModel.core/src/main/java/edu/gemini/spModel/core/Semester.java#L69-L73), by the way.  I agree though that it's not a good time to go about replacing the PIT `Semester` class.